assets/js/index.js

```diff
@@ -0,0 +1,39 @@
+const addIcon = document.getElementById("add-icon");
+const input = document.getElementById("input");
```

Comment:
Input is a very general name. Please be more specific.

Reply:
+ addIcon

Reply:
Done.